chain/default_syncer_test.go

```diff
@@ -893,9 +893,12 @@ func TestTipSetWeightDeep(t *testing.T) {
 	f2b2 := RequireMkFakeChildCore(require,
 		FakeChildParams{Parent: testhelpers.RequireNewTipSet(require, f2b1), GenesisCid: calcGenBlk.Cid(), StateRoot: bootstrapStateRoot, MinerAddr: info.Miners[3].Address},
 		wFun)
+	// This should fix https://github.com/filecoin-project/go-filecoin/issues/1828
```

Comment:
@phritz This last block did not have a Proof and winning Ticket.  It's surprising it didn't fail more often.